core/src/sql/statements/define/function.rs

```diff
@@ -100,36 +98,15 @@ impl fmt::Display for DefineFunctionStatement {
 
 impl InfoStructure for DefineFunctionStatement {
 	fn structure(self) -> Value {
-		let Self {
-			name,
-			args,
-			block,
-			comment,
-			permissions,
-			..
-		} = self;
-		let mut acc = Object::default();
-
-		acc.insert("name".to_string(), name.structure());
-
-		acc.insert(
-			"args".to_string(),
-			Value::Array(
-				args.into_iter()
-					.map(|(n, k)| Value::Array(vec![n.structure(), k.structure()].into()))
-					.collect::<Vec<Value>>()
-					.into(),
-			),
-		);
-
-		acc.insert("block".to_string(), block.structure());
-
-		acc.insert("permissions".to_string(), permissions.structure());
-
-		if let Some(comment) = comment {
-			acc.insert("comment".to_string(), comment.into());
-		}
-
-		Value::Object(acc)
+		Value::from(map! {
```

Comment:
Permissions is missing

Reply:
Good spot! Fixed in [64a3b48](https://github.com/surrealdb/surrealdb/pull/4217/commits/64a3b48ae968786d9be8e67ec0cfc1e9ca308929)